src/rebar_utils.erl

```diff
@@ -928,3 +930,116 @@ is_list_of_strings(List) when is_list(hd(List)) ->
     true;
 is_list_of_strings(List) when is_list(List) ->
     true.
+
+%%------------------------------------------------------------------------------
+%% @doc
+%% Return the SSL options adequate for the project based on
+%% its configuration, including for validation of certs.
+%% @end
+%%------------------------------------------------------------------------------
```

Comment:
Didn't we get that cert check for free by using a dep?

Reply:
Yes, but `rebar_api` has the `ssl_opts` function and I didn't want to remove that. Plus it is used by `rebar_local_upgrade`, but I moved it to `utils` so that the api could still call it too.

Reply:
Understood